src/fmu/ensemble/ensemble.py

```diff
@@ -578,6 +597,46 @@ def get_smrykeys(self, vector_match=None):
                 logger.warning("No EclSum available for realization %d", index)
         return list(result)
 
+    def get_smry_meta(self, column_keys=None):
+        """
+        Provide metadata for summary data vectors.
+
+        A dictionary indexed by summary vector names is returned, and each
+        value is another dictionary with potentially the following metadata types:
+        * unit (string)
+        * is_total (bool)
+        * is_rate (bool)
+        * is_historical (bool)
+        * get_num (int) (only provided if not None)
```

Comment:
Add `keyword` and `wgname` to docstrings (Applies to all get_smry_metadata(), not commenting on the others)
```suggestion
        * get_num (int) (only provided if not None)
        * keyword (str)
        * wgname (str or None)
```